package.json

```diff
@@ -4,14 +4,18 @@
   "private": true,
   "description": "",
   "scripts": {
+    "start": "grunt dev",
```

Comment:
This should not be here, see other comments for why. In summary: performance (bundle generated twice), and unneeded new dependencies